packages/xo-web/src/xo-app/index.js

```diff
@@ -272,6 +279,25 @@ export default class XoApp extends Component {
                   </button>
                 </div>
               )}
+              {isTrialRunning(trial.trial) && !this.state.dismissedTrialBanner && (
+                <div className='alert alert-info mb-0'>
+                  <a
+                    href='https://xen-orchestra.com/#!/xoa?pk_campaign=xo_source_banner'
+                    rel='noopener noreferrer'
+                    target='_blank'
+                  >
```

Comment:
If they see this banner, it means that they're already using XOA so the link is irrelevant.